example/example/ExampleViewController.swift

```diff
@@ -24,15 +24,23 @@ class ExampleViewController: UIViewController {
     }
 
     @IBAction func subscribeAction(sender: AnyObject) {
-
         /* Subscribe to messages on the sport channel */
         channel = client.channels.get("sport")
         channel.subscribe { message in
             print("Received \(message.data)")
         }
     }
 
+    /* Publish a message when the button is pressed */
     @IBAction func publishAction(sender: AnyObject) {
+        channel.publish("update", data: "Man United") { error in
+            guard error == nil else {
+                /* Print an error message when error occurres */
+                print("Message not sent, error occurred: \(error!)")
+                return
+            }
+            /* Print a success message when the message is sent */
+            print("Message successfully sent")
```

Comment:
This is an Xcode app that is intended to run on a device or simulator yet you are printing this output to the console so no one will see this.  The point of the tutorial is to have something interactive they can run and see how it works.  You will need to make sure all `print` statements actually show something to the user in the UI.  Without this, the tutorial when run locally will simply appear to be broken.